Obsidian/Server.cs

```diff
@@ -655,7 +663,7 @@ private async Task LoopAsync()
     /// <summary>
     /// Registers the "obsidian-vanilla" entities and objects.
     /// </summary>
-    /// Might be used for more stuff later so I'll leave this here - tides 
+    /// Might be used for more stuff later so I'll leave this here - tides
```

Comment:
Add the additional slash here as it's not part of documentation.

Reply:
What?

Reply:
Tides forgot to remove the third slash from the comment.